hilti/toolchain/include/ast/ast-context.h

```diff
@@ -212,7 +212,7 @@ class ASTContext : public std::enable_shared_from_this<ASTContext> {
      * dependent units, vs just direct dependencies of the specified unit
      * @return set of dependencies
      */
-    std::vector<declaration::module::UID> dependencies(const declaration::module::UID& uid,
+    std::set<declaration::module::UID> dependencies(const declaration::module::UID& uid,
```

Comment:
We do not care about the order in the set, but pay for that feature. Let's instead use a `std::unordered_set` here and everywhere else.

Reply:
Actually I think we do, at least for consistency/stability in our output (incl. debugging). And these sets are small so I think it shouldn't matter performance-wise.